commands/permissions.go

```diff
@@ -46,11 +49,22 @@ var ShowRoleCmd = &cobra.Command{
 	RunE:       withClient(showRoleCmdF),
 }
 
+var ResetCmd = &cobra.Command{
+	Use:   "reset <role_name>",
+	Short: "Reset default permissions for role (EE Only)",
+	Long:  "Reset the given role's permissions to the set that was originally released with",
+	Example: `  # Reset the permissions of the 'system_read_only_admin' role.
+  permissions reset system_read_only_admin`,
```

Comment:
small, nonblocking nit I missed before:
```suggestion
  $ mmctl permissions reset system_read_only_admin`,
```